### DISCUSSION TITLE
How to get figure of performance comparison？

### DISCUSSION THREAD

**Discussion by XueBaolu** [alibaba/FederatedScope#581] [Q&A] [answered]
The wandb is supposed by FS to log the experiments. But can it be used to draw comparison charts for multiple algorithms? 
If possible, please teach me how to do or share some materials. 
If not, how can I gain results such as test_loss to draw my own figures after the experiments?
Thank you so much.

**Comment by rayrayraykk** [comment]
You might need this: https://docs.wandb.ai/guides/app/features/panels/run-comparer to compare two runs (two algorithms).

**Reply by XueBaolu** [reply]
Thank you! I will try it.

**Comment by XueBaolu** [comment]
Pardon, the link you shared construct to filtrate the different configs among experiments.
However, I want compare performance of different algorithms like this:
![image](https://user-images.githubusercontent.com/90586612/236737406-0cbd115a-ac97-42f2-867b-499c4ae415f3.png)
can I realize it by wandb? If so please give some suggestions. Thanks!

**Comment by rayrayraykk** [comment, marked as answer]
I think you can get this figure by changing the name of the experiment.

```
cfg.expname = ''  # detailed exp name to distinguish different sub-exp
cfg.expname_tag = ''  # detailed exp tag to distinguish different
```

Select the algorithms you want to compare:
![image](https://user-images.githubusercontent.com/39145382/236748165-ddbea9bf-8e03-4878-a4c9-bdb84afd7ccf.png)

The figure you might want:
![image](https://user-images.githubusercontent.com/39145382/236748078-a466cb14-9593-4295-8df2-6ae15e7942cc.png)

**Reply by XueBaolu** [reply]
I get it! Thank you so much
